Atomic/util.py

```diff
@@ -241,15 +242,18 @@ def skopeo_inspect(image, args=None):
 
     cmd = ['skopeo', 'inspect'] + args + [image]
     try:
-        results = subp(cmd)
+        results = subp(cmd, newline=newline)
     except OSError:
         raise ValueError("skopeo must be installed to perform remote inspections")
     if results.return_code is not 0:
         # Need to check if we are dealing with a v1 registry
         check_v1_registry(image)
         raise ValueError("Unable to interact with this registry: {}".format(results.stderr))
     else:
-        return json.loads(results.stdout.decode('utf-8'))
+        if return_json:
+            return json.loads(results.stdout.decode('utf-8'))
+        else:
+            return results.stdout
```

Comment:
A single function returning two so wildly different types?  If anybody ever tried to do static analysis of Python, this would throw if off completely.

shrug* not my codebase, but personally I’d prefer a separate `skopeo_inspect_raw` function (which could in turn hardcode much more). Or, perhaps, just inline the `util.subp` call into `sign.py`.